app/src/hooks/api/useSearchApi.ts

```diff
@@ -40,6 +40,20 @@ const useSearchApi = (axios: AxiosInstance) => {
 
   const getSpatialMetadata = async (submissionSpatialComponentId: number): Promise<ISpatialMetadata> => {
     const { data } = await axios.get(`/api/dwc/spatial/${submissionSpatialComponentId}/metadata`);
+
+    return data;
+  };
+
+  const getSpatialDataFile = async (criteria: {
```

Comment:
Can you add a test for this function.  We just have a really basic test for each of the api functions. We haven't gone super exhaustive with those tests, mostly they just help ensure the URL doesnt get changed unintentionally.

Reply:
I've added a basic test for the new endpoint.